homeassistant/components/aws/notify.py

```diff
@@ -231,3 +234,52 @@ async def async_send_message(self, message="", **kwargs):
 
             if tasks:
                 await asyncio.gather(*tasks)
+
+
+class AWSEventBridge(AWSNotify):
+    """Implement the notification service for the AWS EventBridge service."""
+
+    service = "events"
+
+    async def async_send_message(self, message="", **kwargs):
```

Comment:
Could you add type hints here to enable type checking?

Reply:
The other async_send_message calls in the same file are missing type hints. I'll add type hints to all of them: like so: 
`async def async_send_message(self, message: str = "", **kwargs: Any) -> None:`
(I copied this from one of the other notify integrations)

Reply:
That's the idea, but if you are not feeling comfortable with this change we can add them later.

Reply:
I'm fine with that. I just wasn't sure if you were looking for more accurate types than those so I wanted to confirm.